sfxclient/gometrics.go

```diff
@@ -59,3 +65,47 @@ func (g *goMetrics) Datapoints() []*datapoint.Datapoint {
 		Gauge("num_goroutine", dims, int64(runtime.NumGoroutine())),
 	}
 }
+
+func (g *goEvents) Events() []*event.Event {
```

Comment:
@mdubbyap It provides dummy datapoints and now events for the tests.  It was easier for me to copy, paste, and modify the datapoints rather than write out a whole new list of events.